src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -90,7 +90,7 @@
       <PathToMergedNuGetPack>$(OutputPath)ilmerge\NuGet.Build.Tasks.Pack.dll</PathToMergedNuGetPack>
     </PropertyGroup>
     <ItemGroup>
-      <BuildArtifacts Include="$(OutputPath)*.dll" Exclude="$(OutputPath)*Pack.dll" />
+      <BuildArtifacts Include="$(OutputPath)publish\*.dll" Exclude="$(OutputPath)publish\*Pack.dll" />
```

Comment:
how was this working before? Why do we need this change?

Reply:
Yeah I'm not even sure how I found this.  For `netstandard2.0`, here is the difference between `$(OutputPath)` and `$(OutputPath)\publish`:

```diff
+Microsoft.Extensions.FileProviders.Abstractions.dll
+Microsoft.Extensions.FileSystemGlobbing.dll
+Microsoft.Extensions.Primitives.dll
+Newtonsoft.Json.dll
NuGet.Build.Tasks.Pack.dll
NuGet.Commands.dll
NuGet.Common.dll
NuGet.Configuration.dll
NuGet.Credentials.dll
NuGet.DependencyResolver.Core.dll
NuGet.Frameworks.dll
NuGet.LibraryModel.dll
NuGet.Packaging.dll
NuGet.ProjectModel.dll
NuGet.Protocol.dll
NuGet.Versioning.dll
+System.Buffers.dll
+System.Formats.Asn1.dll
+System.Memory.dll
+System.Numerics.Vectors.dll
+System.Runtime.CompilerServices.Unsafe.dll
+System.Security.Cryptography.Cng.dll
+System.Security.Cryptography.Pkcs.dll
+System.Security.Cryptography.ProtectedData.dll
```

For `net472` folders are the same.  I'm not sure how it was working before, do you want me to dive more into it and figure out what's going on in the shipped .NET SDK?

Reply:
I'm sure tbh. I'm not sure I have a good grasp on the severity of this.